Cargo.toml

```diff
@@ -11,7 +11,7 @@ categories = ["web-programming::http-client"]
 
 [dependencies]
 hyper = "0.10.2"
-hyper-native-tls = "0.2"
+hyper-native-tls = { version = "0.2", optional = true }
```

Comment:
No, not objecting really. Just remembering the joys of when TLS was optional in hyper...

I do kind of wish this was pushed for instead in the native-tls repo, so that native-tls resorts to rustls on Linux instead of OpenSSL...